sdk-base/src/main/java/com/mapbox/maps/plugin/viewport/experimental/transition/DefaultViewportTransition.kt

```diff
@@ -0,0 +1,7 @@
+package com.mapbox.maps.plugin.viewport.experimental.transition
+
+import com.mapbox.maps.plugin.viewport.experimental.data.DefaultViewportTransitionOptions
+
+interface DefaultViewportTransition : ViewportTransition {
```

Comment:
Same here. I don't think we need this interface.

Reply:
As discussed offline, we need interfaces in sdk-base as users won't be able to access to the real implementations without casting. This is a limitation with our plugin architecture, which we should revisit in the next major release.